h2o-core/src/main/java/hex/Model.java

```diff
@@ -937,6 +949,46 @@ public void printTwoDimTables(StringBuilder sb, Object o) {
       printTwoDimTables(sb, this);
       return sb.toString();
     }
+
+    private TwoDimTable[] createReproducibilityInformationTable(ModelBuilder modelBuilder) {
+      TwoDimTable nodeInformation = ReproducibilityInformationUtils.createNodeInformationTable();
+      TwoDimTable clusterConfiguration = ReproducibilityInformationUtils.createClusterConfigurationTable();
+      TwoDimTable inputFramesInformation = createInputFramesInformationTable(modelBuilder);
+      TwoDimTable[] tables = {nodeInformation, clusterConfiguration, inputFramesInformation};
+      return tables;
+    }
+
+    public TwoDimTable createInputFramesInformationTable(ModelBuilder modelBuilder) {
+      List<String> colHeaders = new ArrayList<>();
+      List<String> colTypes = new ArrayList<>();
```

Comment:
this seems like it can be just an array, its never modified and the contents are the same, same for the other two Lists